arangodb-net-standard/CollectionApi/CollectionApiClient.cs

```diff
@@ -108,5 +108,24 @@ public async Task<GetCollectionsResponse> GetCollectionsAsync(GetCollectionsOpti
                 throw await GetApiErrorException(response);
             }
         }
+
+        /// <summary>
+        /// Get a revision of the collection
+        /// /_api/collection/{collection-name}/revision
```

Comment:
It's trivial but can we use punctuation for sentences? (use a dot here)
Also maybe add the HTTP word? like: `GET /_api/collection/{collection-name}/revision`